setup.py

```diff
@@ -513,7 +513,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'jira',
     'jsondiff',
     'mongomock',
-    'moto~=2.2, >=2.2.7',
+    'moto~=2.2, >=2.2.4.dev12',
```

Comment:
moto 2..2.7 is required by kinesis. See: https://github.com/apache/airflow/pull/18337

Reply:
gah.   I messed up a merge there, I'll correct it back to `2.2.7`

Reply:
Should be addressed in https://github.com/apache/airflow/pull/18645/commits/b577f159075d1c6b75d90c54303041b78025f122